omero/sysadmins/troubleshooting.txt

```diff
@@ -324,10 +324,10 @@ mod_wsgi will deadlock if run in daemon mode while mod_python is enabled
 OMERO.web migrating from Apache to Nginx
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Since OMERO 5.2.6 support for Apache and mod_wsgi deployment is deprecated.
-It is recommended to use :doc:`/sysadmins/unix/install-web/install-nginx` or
-:doc:`OMERO.web trial deployment </sysadmins/unix/install-web/install-web-trial>`.
-Official support is likely to be dropped during the 5.3.x line.
+Support for Apache and mod_wsgi deployment was deprecated
+in OMERO 5.2.6. It is recommended to use
+:doc:`/sysadmins/unix/install-web/install-nginx` or
+:doc:`OMERO.web WSGI deployment </sysadmins/unix/install-web/web-deployment>`.
```

Comment:
Better to just remove the set link text and let it automatically use the page title